sccm/osd/get-started/manage-operating-system-images.md

```diff
@@ -72,6 +72,8 @@ Before you can use an OS image, add it to your Configuration Manager site.
 
 5.  Complete the wizard.  
 
+**Powershell Cmd-let** [New-CMOperatingSystemImage](https://github.com/MicrosoftDocs/sccm-docs-powershell-ref/blob/master/sccm-ps/ConfigurationManager/New-CMOperatingSystemImage.md)

```

Comment:
Use the published docs URL instead of the markdown on GitHub: https://docs.microsoft.com/powershell/module/configurationmanager/new-cmoperatingsystemimage?view=sccm-ps (and make sure to not include any locales like en-us in the URL)

Here's a suggestion with that URL along with some additional text to describe it further:

```suggestion
For the PowerShell cmdlet equivalent of this console wizard, see [New-CMOperatingSystemImage](https://docs.microsoft.com/powershell/module/configurationmanager/new-cmoperatingsystemimage?view=sccm-ps).
```

FYI, if you accept my suggestion, note that the automation process may auto close this pull request. But that's ok! I'll reopen and finalize the pull request.